bin/docker/run

```diff
@@ -9,7 +9,7 @@ set -e
 env_vars="$(printenv -0 | sed -zn 's/^\([^=]\+\)=.*$/\1\n/p' | tr -d '\0')"
 travis_vars="USER HOME DEBIAN_FRONTEND \
 	RAILS_ENV RACK_ENV MERB_ENV JRUBY_OPTS JAVA_HOME \
-	CI CONTINUOUS_INTEGRATION $(echo "$env_vars" | grep '^TRAVIS')"
+	CI CONTINUOUS_INTEGRATION $(echo "$env_vars" | { grep '^TRAVIS' || true; })"
```

Comment:
Mmm, this is tricky, beaver assumes travis presence, I think some stuff depends on travis variables, have you checked?